### PR TITLE
ChadoStorage load is working

### PR DESCRIPTION
**Pull request by spficklin**
This PR is for @4ctrl-alt-del to merge the new ChadoStorage class (and Chado property type) classes into his branch to make sure all looks good.

### CODE REVIEW

File: tripal/src/TripalStorage/Interfaces/TripalStorageInterface.php [hunk 1]
```diff
@@ -7,21 +7,21 @@
...
-interface TripalStoragePluginInterface extends PluginInspectionInterface {
+interface TripalStorageInterface extends PluginInspectionInterface {
 
   /**
    * Adds the given array of new property types to this tripal storage plugin.
    *
-   * @param array $schemas
-   *   Array of \Drupal\tripal\Base\StoragePropertyTypeBase objects.
+   * @param array $types
+   *   Array of \Drupal\tripal4\Base\StoragePropertyTypeBase objects.
```
Comment (laceysanderson) [inline tripal/src/TripalStorage/Interfaces/TripalStorageInterface.php]: This path still needs to be updated now it's not in tripal4 directory any more.

Comment (laceysanderson) [reply tripal/src/TripalStorage/Interfaces/TripalStorageInterface.php]: These \Drupal\tripal4\... paths are throughout this file and should be updated :-)

**Review by laceysanderson — COMMENTED** [review]
Just something that jumped out at me :-) This PR is extremely exciting!!! 🎉 
(still clearly needs the review by Josh)

File: tripal/src/TripalStorage/Interfaces/TripalStorageInterface.php (hunk 1 above)
Comment (4ctrl-alt-del) [reply tripal/src/TripalStorage/Interfaces/TripalStorageInterface.php]: That's the only thing I noticed too. Replace tripal4\Base -> tripal\TripalStorage and I think we're good.

**Comment by 4ctrl-alt-del** [comment]
@spficklin Fix the tripal4\Base issue mentioned by Lacey above and we are good to merge.

File: tripal/src/TripalStorage/Interfaces/TripalStorageInterface.php (hunk 1 above)
Comment (spficklin) [reply tripal/src/TripalStorage/Interfaces/TripalStorageInterface.php]: It's fixed now.  It looks like all of those were just in comments.

**Pull request merged**